docs/docs/getting-started/vim-tips.md

```diff
@@ -0,0 +1,45 @@
+# Tips for users coming from Vim
+
+### How do I get the tabs to behave as they do in Vim?
+
+Add these settings to your configuration:
+
+```
+  "oni.layout.singleTabMode": true,
+  "oni.layout.layoutTabPosition": "top",
+```
+
+### How do I replicate my vimrc?
+
+Using the
+[`"experimental.viml"`](https://onivim.github.io/docs/configuration/settings#experimental)
+setting you can already successfully run many VimL commands like keybindings.
+But it is experimental for good reason. Many commands do not work, and there's
+no overview of what does and does not work.
```

Comment:
Hopefully during this milestone we can get this non-experimental and have a clear guide in terms of which `:ex` commands, viml functions, and settings work in Onivim.